.gitignore

```diff
@@ -4,9 +4,5 @@ node_modules
 # coverage
 coverage
 
-# Ignore lock files
-package-lock.json
-yarn.lock
```

Comment:
Removed the yarn.lock .gitignore, apparently this needs to be present for .travis.yml to realize it needs to use yarn.lock